src/RequestHandler.php

```diff
@@ -82,6 +88,7 @@ public function handle(ConnectionInterface $incoming)
         });
 
         $this->start = microtime(true);
+        $this->requestSentAt = microtime(true);
```

Comment:
Cant we use start?

Reply:
I thought about it, but start changes in the verboseTimer function so its not reliable.